e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/submit-deal-cancellation-effective-from-in-future.spec.js

```diff
@@ -119,6 +122,28 @@ context('Deal cancellation - submit cancellation with "effectiveFrom" in future'
         activitiesPage.activitiesTimeline().contains(`Date effective from: ${tomorrow.d_MMMM_yyyy}`);
         activitiesPage.activitiesTimeline().contains(`Comments: -`);
       });
+
+      it('should display the cancellation success banner on each of the other sub navigation tabs', () => {
+        caseSubNavigation.tasksLink().click();
+        successBanner().should('exist');
+        cy.assertText(successBanner(), expectedSuccessBannerText);
+
+        caseSubNavigation.partiesLink().click();
+        successBanner().should('exist');
+        cy.assertText(successBanner(), expectedSuccessBannerText);
+
+        caseSubNavigation.documentsLink().click();
+        successBanner().should('exist');
+        cy.assertText(successBanner(), expectedSuccessBannerText);
+
+        caseSubNavigation.activityLink().click();
+        successBanner().should('exist');
+        cy.assertText(successBanner(), expectedSuccessBannerText);
+
+        caseSubNavigation.underwritingLink().click();
+        successBanner().should('exist');
```

Comment:
No need for the `.should('exist');` checks here - if it doesn't exist, `cy.assertText` will fail because it can't find the text 😎